test/martha/martha_v3.test.js

```diff
@@ -1359,6 +1359,13 @@ test.serial('martha_v3 should parse Data Object uri with the AnVIL prefix dg.ANV
     );
 });
 
+test.serial('martha_v3 should parse Data Object uri with the new AnVIL prefix drs.anv0', (t) => {
+    t.is(
+        determineDrsProviderWrapper('drs://drs.anv0/00008531-03d7-418c-b3d3-b7b22b5381a0'),
```

Comment:
To comply with the current DRS specification, the separator between the compact identifier and the accession id must be a colon character, as shown [here](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.3.0/docs/#tag/DRS-API-Principles/DRS-URIs:~:text=drs%3A//%5Bprovider_code/%5Dnamespace%3Aaccession). This will be the format that TDR uses to mint DRS URIs with compact identifiers.

Prior to DRS standardization, a variety of formats were used, including the use of a slash separator instead of a colon, and these formats have continued to be supported by Martha, so existing DRS URIs in user workspaces continue to be resolvable.

Net net: For this test, I propose changing:
```
determineDrsProviderWrapper('drs://drs.anv0/00008531-03d7-418c-b3d3-b7b22b5381a0')
```
to:
```
determineDrsProviderWrapper('drs://drs.anv0:00008531-03d7-418c-b3d3-b7b22b5381a0')
```

Reply:
Makes sense, updated with your suggestion!